models/canadarm/urdf/SSRMS_Canadarm2.urdf.xacro

```diff
@@ -0,0 +1,352 @@
+<?xml version="1.0" ?>
+
+<!-- URDF Created from information in the following Paper:
+
+Nguyen, Phung Khac, and Peter C. Hughes. "Teleoperation: from the space shuttle to the space station." Progress in astronautics and aeronautics 161 (1994): 353-353.
+
+The link CoM locations (with respect to link frame) are set to zero as the inertia tensor is defined with respect to the link frame in the paper.
+
+ -->
+<robot name="SSRMS_Canadarm2" xmlns:xacro="http://wiki.ros.org/xacro">
+
+  <xacro:arg name="xyz" default="0.5 0.0 1.5" />
+  <xacro:arg name="rpy" default="3.1416 0.0 0.0" />
+
+  <!-- Gazebo - ROS2 Control macro -->
+  <xacro:include filename="$(find simulation)/models/canadarm/urdf/SSRMS_Canadarm2.gazebo.xacro" />
+  <xacro:ssrms_canadarm2_gazebo command_interface="position" />
+  
+  <link name="world" />
+  
+  <joint name="world_joint" type="fixed">
+    <parent link="world"/>
+    <child link="Base_SSRMS"/>
+    <!-- Rotate Frame to Match the frames on Fig. 13 in Paper -->
+    <origin rpy="$(arg rpy)" xyz="$(arg xyz)"/>
+  </joint>
+
+  <!-- SSRMS_Canadarm2 Dummy Base Body-->
+  <link name="Base_SSRMS">
+    <inertial>
+      <origin rpy="0 0 0" xyz="0 0 0.5"/>
+      <mass value="243.66"/>
+      <inertia ixx="44.413" ixy="0" ixz="0" iyy="44.413" iyz="0" izz="9.336"/>
+    </inertial>
+    <visual>
+      <origin rpy="0 0 0" xyz="0 0 0"/>
+      <geometry>
+        <mesh filename="$(find simulation)/models/canadarm/meshes/ee.dae" scale="1.0 1.0 1.0" />
+      </geometry>
+    </visual>
+    <collision>
+      <origin rpy="0 0 0" xyz="0 0 0"/>
+      <geometry>
+        <mesh filename="$(find simulation)/models/canadarm/meshes/ee.stl" scale="1.0 1.0 1.0" />
+      </geometry>
+    </collision>
+  </link>
+
+  <!-- Dummy Joint Joining Base to Body B1 -->
+  <joint name="Base_Joint" type="revolute">
+    <parent link="Base_SSRMS"/>
+    <child link="B1"/>
+    <!-- Rotate Frame to Match the frames on Fig. 13 in Paper -->
+    <origin rpy="0 0 0" xyz="0 0 0"/>
+    <axis xyz="0 0 1"/>
+    <limit lower="-4.71239" upper="4.71239" effort="2332" velocity="0.0698132"/>
+  </joint>
+
+
+  <!--Manipulator-->
+  <!-- Link 1 -->
+  <link name="B1">
+    <inertial>
+      <origin rpy="0 -1.5708 0" xyz="0.25082 0 -0.175"/>
+      <mass value="105.98"/>
+      <inertia ixx="12.19" ixy="0" ixz="0" iyy="12.19" iyz="0" izz="3.061"/>
+    </inertial>
+
+    <visual>
+      <origin rpy="0 -1.5708 0" xyz="0.25082 0 -0.175"/>
+      <geometry>
+        <mesh filename="$(find simulation)/models/canadarm/meshes/joint_v3_0.dae" scale="1.0 1.0 1.0" />
+      </geometry>
+    </visual>
+    <collision>
+      <origin rpy="0 -1.5708 0" xyz="0.25082 0 -0.175"/>
+      <geometry>
+        <!--<mesh filename="$(find simulation)/models/canadarm/meshes/joint_combine_link3.stl" scale="1.0 1.0 1.0" />-->
```

Comment:
```suggestion
```

Reply:
Deleted the commented out line, per reviewer's suggestion